### PR TITLE
request response ui feedback (Snackbar)

### PR DESCRIPTION
**Pull request by juanga13**
# Description

Please include a summary of the change.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How has this been tested?

Please describe the tests that you ran to verify your changes.

- [ ] Automation testing (Unit or integration tests)
- [ ] Manual testing (Postman)

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] My pull request is up to date with develop

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -12,164 +12,164 @@ import { isLoggedIn } from "../../services/AuthService";
...
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+    inputRoot: {
+        color: 'inherit',
+    },
+    inputInput: {
+        padding: theme.spacing(1, 1, 1, 0),
+        // vertical padding + font size from searchIcon
```
Comment (brunodeluk) [inline src/components/Header/Header.tsx:162]: Remover comentario

File: src/scenes/session/Register/Register.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            })
+            .catch((error) => {
+                this.setState({ registerStatus: RequestStatus.ERROR, error });
+                this.props.registerCallback(RequestStatus.ERROR, RequestStatus.ERROR);
+            });
+    }
+
+    render() {
+        const { registerStatus, error } = this.state;
+        console.log('render reigsre', this.state);
```
Comment (matiasvenditti) [inline src/scenes/session/Register/Register.tsx:53]: Saquemos este console log

File: src/components/Header/Header.tsx [hunk 3]
```diff
@@ -12,164 +12,164 @@ import { isLoggedIn } from "../../services/AuthService";
...
-							/>
-						</div>
-						{this.renderButtons()}
-					</Toolbar>
-				</AppBar>
-			</div>
-		)
-	}
+    render() {
+        console.log('render headerrrr');
```
Comment (matiasvenditti) [inline src/components/Header/Header.tsx:48]: Console log

File: src/components/Header/Header.tsx [hunk 4]
```diff
@@ -12,164 +12,164 @@ import { isLoggedIn } from "../../services/AuthService";
...
-						<Button onClick={() => this.props.history.push('/signup')}>Registrarte</Button>
-					</ButtonGroup>
-				</div>
-			);
-		}
-	}
+    renderButtons() {
+        const logged = isLoggedIn();
+        if (logged) {
+            console.log('LOGGED IN, RENDER COSITO')
```
Comment (matiasvenditti) [inline src/components/Header/Header.tsx:80]: sacar console log

File: src/components/Header/Header.tsx [hunk 5]
```diff
@@ -12,164 +12,164 @@ import { isLoggedIn } from "../../services/AuthService";
...
+                        open={Boolean(this.state.anchorEl)}>
+                        <MenuItem onClick={() => { this.props.history.push('/profile'); this.handleClose() }}>Ver Perfil</MenuItem>
+                        <MenuItem onClick={() => { this.props.history.push('/'); this.handleClose() }}>Ver Reseñas</MenuItem>
+                        <MenuItem onClick={() => { this.props.history.push('/'); this.handleClose() }}>Crear Autor</MenuItem>
+                        <MenuItem onClick={this.handleLogout}>Cerrar Sesión</MenuItem>
+                    </Menu>
+                </div>
+            )
+        } else {
+            console.log('NOT LOGGED IN, RENDER LOGIN/SINGUP')
```
Comment (matiasvenditti) [inline src/components/Header/Header.tsx:108]: sacar console log

File: src/scenes/session/Register/Register.tsx (hunk 2 above)
Comment (brunodeluk) [inline src/scenes/session/Register/Register.tsx:53]: Sacar este `console.log()`.

**Pull request closed without merging**